R/framework_version.R

```diff
@@ -0,0 +1,4 @@
+framework_version <- function()
+{
+	.Call(R_framework_version)
+}
```

Comment:
Right now I am planning to call this function internally as part of the current `BioCro:::check_out_module()` function. When checking out a module from a library, the framework version used by the two different R packages would be compared. If there is a mismatch, there would be a warning message advising the user to update the R package with the older version, and if that doesn't fix the issue, to contact the maintainer of the R package with the older version. I think this would be a relatively simple way to help make sure new versions of the C++ framework are eventually incorporated into module library packages.

As you suggested, there may also be a case in the future where a new feature gets added to the C++ framework, and this function would be helpful for enforcing a minimum required version.